src/engine/Server.h

```diff
@@ -30,8 +31,9 @@ using std::vector;
 //! The HTTP Server used.
 class Server {
  public:
-  explicit Server(unsigned short port, int numThreads, size_t maxMemGB,
-                  std::string accessToken, bool usePatternTrick = true);
+  explicit Server(unsigned short port, int numThreads,
+                  ad_utility::MemorySize maxMemGB, std::string accessToken,
```

Comment:
The name doesn't need to contain `GB` anymore, so just `maxMemoryUsage` or something.

Reply:
Done, see [04586db](https://github.com/ad-freiburg/qlever/pull/1106/commits/04586dbff18395fde3e0210b8dd1322a46b4f56b).